backend/app.py

```diff
@@ -6,26 +6,33 @@
 
 from utils.api_utils import PrefixMiddleware
 
-
-prefix = os.environ.get('JUPYTERHUB_SERVICE_PREFIX')
-content_database_string = os.environ.get('CONTENT_DATABASE_STRING')
-
 db = SQLAlchemy()
 
+
 def create_app():
     app = Flask(__name__)
+
+    # all URLs will be prefixed with the jupyterhub hub URL
+    prefix = os.environ.get("JUPYTERHUB_SERVICE_PREFIX")
```

Comment:
Would it make sense to test here if this variable is set and raise an error if not?

Reply:
If it is not set, we will use "" as the prefix (so nothing), which is very convenient for testing :-)